src/main/routes/routes.ts

```diff
@@ -98,7 +98,7 @@ export default function(app: Application): void {
   // TODO: expose route only if not on the production environment
   app.get('/mock-session', app.locals.container.cradle.mockSessionController.get);
   app.post('/mock-login', passport.authenticate(authType, { failureRedirect: '/not-found'}),
-    (req, res) => {res.redirect('/');});
+    (req, res) => {res.redirect('/alphabetical-search');});
```

Comment:
Just double checking if this is correct? The link in the ticket looks like it goes to the prototype app